front/src/views/routes/common/cms/CommonCMSPage.vue

```diff
@@ -1210,11 +1213,18 @@ export default {
   .lck-container {
     box-shadow: unset;
     background-color: #FFFFFF;
+    display: block !important;
   }
 
   .lck-container > div {
-    display: block !important;
     height: auto;
+    box-shadow: unset;
+  }
+
+  .lck-container > :not(.lck-container-active) ::v-deep .map-container {
```

Comment:
Could you explain why only the containers that are inactive ?
Could you add some visual tests for these use cases ?

Reply:
> Could you explain why only the containers that are inactive?

Case for page with anchor or tabs.

The initial problem: When printing the pdf, the position of the scale and attribution is shifted in regard to the map.

Because of css in DataRecord.

Map container takes the same size (width) as the other fields. Therefore, the map canvas is updated.

But as the other containers are set to `display: none`, the map canvas is set to the default size. So, when switching to pdf print mode, the map container is resized to the size defined by DataRecord but the canvas is not resized.

The solution is therefore for the active container to display the map with a size adapted to DataRecord (because if you reduce the size, the map is no longer correct) and for the other containers to have a reduced size corresponding to the default one.

>  Could you add some visual tests for these use cases?

Storybook allow to take snapshot with print css?

Reply:
Great, I understand better.
I think we can do it by specifying new devices, but I didn't find if we could create a "print" one.
So, maybe just add comments in the code to remember why you did this ?

Reply:
Done